third_party/CMakeLists.txt

```diff
@@ -368,3 +368,17 @@ target_compile_options(
   -Wno-parentheses-equality
 )
 endif()
+
+# Call Makefile of jcch-dbgen, copy dbgen and qgen into the current build folder, then clean up
+add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/jcch-dbgen/dbgen ${CMAKE_CURRENT_BINARY_DIR}/jcch-dbgen/qgen ${CMAKE_CURRENT_BINARY_DIR}/jcch-dbgen/dists.dss
```

Comment:
The build failure is caused because this is not concurrency-safe. Use the same setup as jemalloc? Shouldn't block the review of the rest.